create_database.py

```diff
@@ -0,0 +1,136 @@
+"""Read in data from a text file. Manipulate the data and insert it
+into an SQLite database with a 'users table' that has columns:
+userID, given_name, family_name
+and a 'cars table' that has columns:
+car, owner
+Return a completion message upon completion.
+"""
+import sqlite3
+import name
+import car
+import read_file
+import user_id
+
+def unpack_data(data_package):
```

Comment:
anywhere you see 'data', you should assume the name is not descriptive. try and remove that word